packages/gatsby/src/utils/develop.js

```diff
@@ -108,7 +108,7 @@ async function startServer(program) {
   // Render an HTML page and serve it.
   app.use((req, res, next) => {
     const parsedPath = parsePath(req.originalUrl)
-    if (parsedPath.extname === `` || parsedPath.extname === `.html`) {
+    if (parsedPath.extname === `` || parsedPath.extname.indexOf(`.html`) === 0) {
```

Comment:
yeah, we are slowly trying reach some sort level consistency on that stuff. Thanks for the change, i realize it's small and a bit inane to have asked 🙇